engine/execution/computation/computer/computer.go

```diff
@@ -27,6 +27,7 @@ import (
 )
 
 const SystemChunkEventCollectionMaxSize = 256_000_000 // ~256MB
+const SystemChunkLedgerIntractionLimit = 500_000_000  // ~500MB
```

Comment:
@jordanschalm is this enough in comparison to the current 20MB one?

Reply:
I'm pretty sure that would be enough, but we don't know what the actual usage is (since the tx reverts once it crosses the current limit).

Reply:
how much bigger is mainnet in comparison to testnet? (in terms of input size (probably node counts) 10X ?

Reply:
Testnet has ~100 nodes and Mainnet has ~400. But there are also many delegators on Mainnet (and none or close to none on Testnet). I don't know exactly how many delegators there are, but I think there are tens of thousands. I remember seeing just one node (an outlier, but still) with 13,000 delegators.

Reply:
and the system chunk transaction would touch the delegators list?

Reply:
It will when we `moveTokens`, which happens both when we pay rewards (service account) and when we transition epochs (system chunk)

Reply:
do we remember what this use to be before we brought it down to 20mb?

Reply:
2_000_000_000 (~2GB) initially but I don't think we ever reached it with the old version

Reply:
Since this is the only transaction in the system chunk, I'm going to change it to 1GB (and leave 1GB for proofs)

Reply:
sounds good